lib/scss_lint/plugins/linter_dir.rb

```diff
@@ -19,6 +19,29 @@ def load
       def ruby_files
         Dir.glob(File.expand_path(File.join(@dir, '**', '*.rb')))
       end
+
+      # Returns the {SCSSLint::Config} for this directory.
+      #
+      # This is intended to be merged with the configuration that loaded this
+      # plugin.
```

Comment:
This comment confuses me a little since we use `merge_with_default: false` below. What is indented to merge, and where does that happen? I don't have a lot of context on how scss-lint configuration works, so excuse me if I'm missing something obvious.